doc/build-instructions/Build_Instructions_V11.md

```diff
@@ -784,3 +785,356 @@ JCL      - d247952 based on jdk-11.0.6+6)
+```
+git clone https://github.com/ibmruntimes/openj9-openjdk-jdk11.git
+```
+
+[2] Change directory into the cloned repository when the process is complete:
+```
+cd openj9-openjdk-jdk11
+```
+:bulb:
+Cloning this repository can take a while due to the huge size of the OpenJDK project
+
+[3] Now fetch additional sources from the Eclipse OpenJ9 project and its clone of Eclipse OMR:
+```
+bash get_source.sh
+```
+
+### 5. Configure the cross-compiling environment
+:rocket:
+When you have all the source files on the host system, run the following configure command to set up the cross-compilation environment:
+```
+export JAVA_HOME=/<path_to_boot_JDK>
+export RISCV64=/<path_to_gnu_cross_toolchain>
+export PATH="$RISCV64/bin:$JAVA_HOME/bin:$PATH"
+
+bash configure --disable-warnings-as-errors \ 
+               --disable-ddr \
+               --with-boot-jdk=/<path_to_boot_JDK> \
+               --with-build-jdk=/<path_to_boot_JDK> \
+               --openjdk-target=riscv64-unknown-linux-gnu \
+               --with-sysroot=/<path_to_your_fedora_mount_directory> \
+               --with-freemarker-jar=/<path_to>/freemarker.jar
+```
+
+:pencil:
+If you want to build an OpenJDK that uses OpenSSL, you must specify `--with-openssl={path_to_library}`
+ where:
+  - `path_to_library` uses an OpenSSL v1 library installed on the mounted Fedora OS image.
+e.g.
+```
+--with-openssl=/<path_to_your_fedora_mount_directory>/usr
+```
+
+You can check the version of OpenSSL on your target system as follows:
+```
+<your_fedora_mount_directory>/usr/bin# file openssl
+openssl: ELF 64-bit LSB executable, UCB RISC-V, version 1 (SYSV), 
+dynamically linked, interpreter /lib/ld-linux-riscv64-lp64d.so.1, 
+for GNU/Linux 4.15.0, BuildID[sha1]=715cfde5a3bdc6ff31b6cc2e449f06df1b3c465a, not stripped
+```
+
+### 6. Cross-build the JDK
+:rocket:
+Now you're ready to cross-build OpenJDK with OpenJ9:
+```
+make all
+```
+
+A binary for the full developer kit (JDK without DDR support) is built and stored in the following directory:
+
+- **build/linux-riscv64-normal-server-release/images/jdk**
+
+Copy the JDK to a specified directory on the mounted Fedora OS image
+e.g.
+```
+cp -r jdk /<path_to>/<your_fedora_mount_directory>/home/<your_user_name>/new_dir
+```
+:bulb:
+Instead of copying the JDK into the Fedora OS image, you can also zip the JDK & upload it onto Fedora via the `scp`  command as follows:
+```
+[root@stage4 home/<your_user_name>/new_dir]# scp  <user>@<the_IP_address_on_your_host_system>:/<path_to>/jdk.zip .
+[root@stage4 home/<your_user_name>/new_dir]# unzip -qq jdk.zip
+[root@stage4 home/<your_user_name>/new_dir]# jdk/bin/java -version
+```
+
+### 7. Test the JDK on Fedora/QEMU
+:rocket:
+Unmount the Fedora OS image after the cross-compilation is complete.
+```
+sudo umount /<path_to_your_fedora_mount_directory>
+```
+
+Boot Fedora via QEMU with the command `qemu-system-riscv64` at **2** and change to your jdk directory on Fedora:
```

Comment:
I suggest this would be easier to follow if it made reference to 'step 2 above' rather than just `**2**`.

Reply:
Fixed.

Reply:
I like the link, but it should not be capitalized ('step 2', not 'Step 2').

Reply:
Fixed.